lib/ui/signin/sign_in_form.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter_ic/extension/context_extension.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
+import 'package:survey_flutter_ic/widget/flat_button_text.dart';
+import 'package:survey_flutter_ic/widget/text_input_field.dart';
+
+class SignInForm extends StatefulWidget {
+  const SignInForm({Key? key}) : super(key: key);
+
+  @override
+  State<SignInForm> createState() => _SignInFormState();
+}
+
+class _SignInFormState extends State<SignInForm> {
+  bool _isSignInButtonEnabled = false;
+  String? _emailInput;
+  String? _passwordInput;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        TextInputField(
+          hintText: context.localization.sign_in_email_label,
+          textInputType: TextInputType.emailAddress,
+          onChanged: (input) => {
+            setState(() {
+              _emailInput = input;
+              _validateInputFields();
+            })
+          },
+          onSubmitted: (input) => {
+            setState(() {
+              _emailInput = input;
+              _validateInputFields();
+            })
+          },
+        ),
+        const SizedBox(height: space20),
+        TextInputField(
+          hintText: context.localization.sign_in_password_label,
+          isObscureText: true,
+          obscuringCharacter: "●",
```

Comment:
Can we remove `obscuringCharacter: "●"`? It seems it works properly by setting this `isObscureText: true,

Reply:
@Wadeewee Good catch 👍 We handled the case `obscuringCharacter ` is empty with the default character in the `TextInputField` widget, so we could get rid of this.